c/pthread-lit/fkp2013_false-unreach-call.i

```diff
@@ -654,12 +654,9 @@ void __VERIFIER_assert(int cond) {
   }
   return;
 }
-void __VERIFIER_atomic_assert(int cond) {
```

Comment:
Got it, makes sense. The only thing you have to be careful about when removing `__VERIFIER_atomic_assert` is that `__VERIFIER_assert` is already defined in these benchmarks. If it is not, then you have to define it.